GameOfLifeApp/Services/Serializer.cs

```diff
@@ -31,14 +31,39 @@ public void SaveData(IGameField field)
         public GameField Load()
         {
             GameField field = new GameField();
-            
+
             using (StreamReader sr = File.OpenText(_fileName))
+            {
+                string fileText = sr.ReadToEnd();
+                field = JsonConvert.DeserializeObject<GameField>(fileText);
+            }
+
+            return field;
+        }
+
+        /// <summary>
+        /// Creates a StreamWriter and adds some text to the writer using StreamWriter.
+        /// </summary>
+        /// <param name="games">Saves the data of the object 'games'.</param>
+        /// <param name="field">Saves the data of the object 'field'.</param>
+        public void SaveAllGames(List<IGameField> games, IGameField field)
+        {
+            using (StreamWriter writer = File.CreateText(_fileName))
+
+            {
+                string serializedGames = JsonConvert.SerializeObject(games);
+                string serializedCountOfIteration = JsonConvert.SerializeObject(field.CountIteration);
```

Comment:
Seems like variable is declared & set but never used.

Reply:
Removed.